app/models/case_log.rb

```diff
@@ -294,6 +301,13 @@ def get_inferred_la(postcode)
     end
   end
 
+  def get_has_benefits
+    return "yes" if ["1. Housing benefit",
```

Comment:
Does this need to be "Yes"?

Reply:
Array itself might also be worth extracting out into a named constant?

Reply:
Any particular reason not to make `has_benefits` an enum?

Reply:
I think this is just for us to track, but yeah, makes sense to do it the way we do it everywhere else